src/scancode/cli.py

```diff
@@ -231,11 +231,46 @@ class ScanCommand(BaseCommand):
     short_usage_help = '''
 Try 'scancode --help' for help on options and arguments.'''
 
-    def get_params(self, ctx):
-        """
-        Add options returned by plugins to the params list
-        """
-        return super(BaseCommand, self).get_params(ctx)
+    def format_options(self, ctx, formatter):
+        """Writes all the option groups into the formatter if they exist."""
+        groups = OrderedDict([
+            ('scans', []),
+            ('output', []),
+            ('pre-scan', []),
+            ('post-scan', []),
+            ('misc', []),
+            ('core', []),
+        ])
+
+        for param in self.get_params(ctx):
+            help_record = param.get_help_record(ctx)
+            if help_record is not None:
```

Comment:
That striked strange to me as well, but I picked up the code directly from click's core and modified it, hence I thought best not to mess with that `if`.